build/import/Packages.targets

```diff
@@ -124,19 +125,20 @@
     <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Analyzers"                         Version="16.4.28-pre" />
 
     <!-- Roslyn -->
-    <PackageReference Update="Microsoft.VisualStudio.LanguageServices"                                Version="3.2.0-beta4-19359-03" />
-    <PackageReference Update="Microsoft.CodeAnalysis"                                                 Version="3.2.0-beta4-19359-03" />
+    <PackageReference Update="Microsoft.VisualStudio.LanguageServices"                                Version="3.4.0" />
+    <PackageReference Update="Microsoft.CodeAnalysis.Workspaces.Common"                               Version="3.4.0" />
+    <PackageReference Update="Microsoft.CodeAnalysis"                                                 Version="3.4.0" />
     <PackageReference Update="Microsoft.VisualStudio.IntegrationTest.Utilities"                       Version="2.6.0-beta1-62113-02" />
 
     <!-- Analyzers -->
     <!-- Note: analyzers from github.com/dotnet/roslyn-analyzers are built together and share the same version -->
-    <PackageReference Update="Microsoft.CodeAnalysis.Analyzers"                                       Version="2.9.4" />
-    <PackageReference Update="Microsoft.CodeQuality.Analyzers"                                        Version="2.9.4" />
-    <PackageReference Update="Microsoft.NetFramework.Analyzers"                                       Version="2.9.4" />
-    <PackageReference Update="Microsoft.NetCore.Analyzers"                                            Version="2.9.4" />
-    <PackageReference Update="Microsoft.CodeAnalysis.BannedApiAnalyzers"                              Version="2.9.4" />
-    <PackageReference Update="Microsoft.CodeAnalysis.PublicApiAnalyzers"                              Version="2.9.4" />
-    <PackageReference Update="Roslyn.Diagnostics.Analyzers"                                           Version="2.9.4" />
+    <PackageReference Update="Microsoft.CodeAnalysis.Analyzers"                                       Version="2.9.6" />
+    <PackageReference Update="Microsoft.CodeQuality.Analyzers"                                        Version="2.9.6" />
+    <PackageReference Update="Microsoft.NetFramework.Analyzers"                                       Version="2.9.6" />
+    <PackageReference Update="Microsoft.NetCore.Analyzers"                                            Version="2.9.6" />
+    <PackageReference Update="Microsoft.CodeAnalysis.BannedApiAnalyzers"                              Version="2.9.6" />
+    <PackageReference Update="Microsoft.CodeAnalysis.PublicApiAnalyzers"                              Version="2.9.6" />
+    <PackageReference Update="Roslyn.Diagnostics.Analyzers"                                           Version="2.9.6" />
```

Comment:
2.9.7 is the latest release. Any reason not to use it?

Reply:
No, I just bumped everything to the minimum value to avoid package downgrade warnings.